R/linear-mixed-model-deployment.R

```diff
@@ -21,7 +21,6 @@
 #' @param grainCol Optional. The dataframe's column that has IDs pertaining to 
 #' the grain. No ID columns are truly needed for this step.
 #' @param personCol The data frame's columns that represents the patient/person
-#' @param testWindowCol (depreciated) All data now receives a prediction
 #' @param predictedCol Column that you want to predict. If you're doing
 #' classification then this should be Y/N.
 #' @param impute For training df, set all-column imputation to F or T.
```

Comment:
Let the user know that impute uses the values from the model that was saved in the development step.